wiki/Tournaments/IntlConvertCup/2024/en.md

```diff
@@ -0,0 +1,550 @@
+| Upper | **OwOgres** | **5** | 1 | Colo-Colo | [#1](https://osu.ppy.sh/community/matches/116439131) |
+| Upper | **Capo so cute v3** | **5** | 0 | Sightread Gaming | [#1](https://osu.ppy.sh/community/matches/116435735) |
+| Upper | **Steam Happy** | **5** | 0 | Ciel Enchanteur | *win by default* |
+| Upper | **Maid Girl UwU** | **5** | 0 | pending | *win by default* |
+
+### Qualifiers
+
+Detailed information on the standings of the Qualifier stage can be found in [the following spreadsheet](https://docs.google.com/spreadsheets/d/1Fut4e4ABYano1sYGa6wokzx9kqEFaVwyfaufN1YzoZU/edit?usp=sharing).
+
+| Seed | Team name | Avg. rank |
+| :-: | :-- | --: |
+| 1 | naruto when | 2.63 |
+| 2 | common madcin L | 3.00 |
+| 3 | Capo so cute v3 | 4.25 |
+| 4 | Red Hawk | 4.50 |
+| 5 | OwOgres | 5.63 |
+| 6 | Steam Happy | 5.88 |
+| 7 | EUphoria | 7.38 |
+| 8 | you tell me | 8.75 |
+| 9 | Haters in Abyss | 10.75 |
+| 10 | :cat: | 11.75 |
+| 11 | GiGiLuNi | 12.00 |
+| 12 | :tac: | 12.50 |
+| 13 | Jump the Beat | 14.63 |
+| 14 | Maid Girl UwU | 15.63 |
+| 15 | OldCloudPlayers | 16.75 |
+| 16 | D. Continental | 17.25 |
+| 17 | +4 ARAM | 17.50 |
+| 18 | ctb = danmaku | 18.13 |
+| 19 | pending | 18.50 |
+| 20 | NSMBW | 19.13 |
+| 21 | Mr. Ladybug | 20.00 |
+| 22 | Seed #33 | 21.25 |
+| 23 | NoApple4 | 21.75 |
+| 24 | PICK DT2!!1!!11 | 23.50 |
+| 25 | Barely Managing | 23.75 |
+| 26 | turu | 26.38 |
+| 27 | Ciel Enchanteur | 26.63 |
+| 28 | Colo-Colo | 28.38 |
+| 29 | Bashame X Osaka | 28.63 |
+| 30 | Sightread Gaming | 28.75 |
+| 31 | Convert noob | 29.50 |
+| 32 | Fate | 29.75 |
+| 33 | Enterprise | 31.25 |
+| 34 | For fun gamers | 31.38 |
+| 35 | 3 dif digits | 33.63 |
+| 36 | looking for a team name | 35.00 |
+| N/A | abubeshki | *DNP* |
+| N/A | Albacete x ICC | *DNP* |
+
+## Ruleset
+
+### General rules
+
+1. Match lobbies across the tournament will adhere to the following room settings:
+   - Team Mode: `Head-to-head`
+   - Win Condition: [`ScoreV2`](/wiki/Gameplay/Score#scorev2)
+2. The mappools for each round will be announced by the tournament management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the tournament management. If there are any teams who are unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the `#reschedule` channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a team does not show up or is unable to field in a sizeable 2v2 roster within **10 minutes** of the start time, their opponent gets to win by default.
+6. If no staff or referee is available, the match will be postponed.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```

Comment:
since these are spaced elsewhere in this file

```suggestion
7. **No Fail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```